packages/material-ui/src/styles/createMixins.js

```diff
@@ -1,13 +1,16 @@
+export const SM_UP_GUTTER_MULTIPLIER = 3;
+export const DEFAULT_GUTTER_MULTIPLIER = 2;
```

Comment:
I would add those as options to createMixins.

Reply:
As part of this? Seems out of scope for fixing the issue?

Reply:
You're totally right. I would still prefer to not export them and hardcode them in the test. This way we have a clear indicator that we would introduce a breaking change if we change those values.